### ISSUE TITLE
More and more follower crawlers

### ISSUE THREAD

**Issue by willemdh** [isaacs/github#533]
I've been noticing I had some followers who came and left again some days later. After doing some fast research why someone would do this, I found tools like these: https://github.com/imraazrally/GreedyGit 

Which enable GitHub users to quickly start following a lot of users. They then wait for a few days and unfollow you again in the hope some of the x users the follow, follow them back. Ways to counter this: Only allow users to follow max 3-5 other users a day? Why would I want to follow 500 new users in one day... This messes up the GitHub experience imho.

Grtz

**Comment by cirosantilli** [comment]
+1 for thinking about it. There was also recently the star bot: https://news.ycombinator.com/item?id=10517527

Spam, oh hard to fight spam.

I'd rather have a rule like: if I'm the n-th person followed / starred on a day, don't show a notification. It annoys me a lot to have action throttles like on Stack Overflow.

Or go hardcore do some kind of google page rank evaluation of people and cut low ranks from notifications :-)

**Comment by QuantumInformation** [comment]
Yeah this is so annoying, we should report users who follow 2k or more or smth, and ban them
